cypress/e2e/occ.cy.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * SPDX-FileCopyrightText: 2025 Nextcloud GmbH and Nextcloud contributors
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+
+describe('Run occ command', function() {
+	it('returns the output', function() {
+		cy.runOccCommand('app:list').then(output => {
+			expect(output.stdout).to.contain('viewer')
```

Comment:
Funny... passes for me locally. Looks like something does not work during the setup of viewer on CI:

![grafik](https://github.com/user-attachments/assets/a6a31f3e-7328-43d4-9acb-7c7b5579afb9)

This seems unrelated to the PR so I will change this to check for `files` and `settings` which should always be around.

Reply:
Looks like github sets an environment variable `BRANCH` that matches the target branch of the PR - but we use it as the server branch for the container.
![grafik](https://github.com/user-attachments/assets/8b6c08b8-69bf-45a8-a7f1-c33053b23a96)